### DISCUSSION TITLE
Can I generate a color map for the results from the command line？

### DISCUSSION THREAD

**Discussion by mshmoon** [Universite-Gustave-Eiffel/NoiseModelling#558] [Q&A]
Can I generate a color map for the results from the command line？

**Comment by gpetit** [comment]
Hi,

NoiseModelling can "only" generate the resulting geometries. If you want to apply a style (like https://noisemodelling.readthedocs.io/en/latest/Noise_Map_Color_Scheme.html), you will have to do it manually (e.g in a GIS app - see https://noisemodelling.readthedocs.io/en/latest/Noise_Map_From_Point_Source.html#generate-noise-level-isosurfaces).

You can probably automatize it, thanks to groovy or other language but you will have to use a dedicated librairies to map the geometries with the appropriate style (FYI they do this kind of things in GeoClimate - https://github.com/orbisgis/geoclimate/  @ebocher )
